Sources/FacebookInstance.swift

```diff
@@ -23,6 +24,7 @@ public protocol FacebookCommand {
     // Settings
     func setAutoLogAppEventsEnabled(_ enabled: Bool)
     func enableAdvertiserIDCollection(_ enabled: Bool)
+    func checkAdvertiserTracking()
```

Comment:
`checkAdvertiserTracking` has to be added to the `MockFacebookInstance` class otherwise tests won’t run